dask/dataframe/io/tests/test_json.py

```diff
@@ -13,6 +13,74 @@
 ddf = dd.from_pandas(df, npartitions=2)
 
 
+@pytest.mark.parametrize("orient", ["split", "records", "index", "columns", "values"])
+def test_read_json_with_path_column(orient):
+    with tmpfile("json") as f:
+        df.to_json(f, orient=orient, lines=False)
+        actual = dd.read_json(f, orient=orient, lines=False, include_path_column=True)
+        actual_pd = pd.read_json(f, orient=orient, lines=False)
+        actual_pd["path"] = pd.Series((str(f),) * len(actual_pd), dtype="category")
+        out = actual.compute()
+        assert_eq(out, actual_pd)
+
+
+def test_read_json_path_column_present():
+    with tmpfile("json") as f:
+        df.to_json(f, orient="records", lines=False)
+        with pytest.raises(ValueError, match="Files already contain"):
+            dd.read_json(f, orient="records", lines=False, include_path_column="x")
+
+
+def test_read_json_with_converter():
+    with tmpfile("json") as f:
+        df.to_json(f, orient="records", lines=False)
+        actual = dd.read_json(
+            f,
+            orient="records",
+            lines=False,
+            include_path_column="path",
+            path_converter=lambda x: "asdf.json",
+        )
+        actual_pd = pd.read_json(f, orient="records", lines=False)
+        actual_pd["path"] = pd.Series(("asdf.json",) * len(actual_pd), dtype="category")
+        out = actual.compute()
+        assert_eq(out, actual_pd)
+
+
+def test_read_orient_not_records_and_lines():
+    with pytest.raises(ValueError, match="Line-delimited JSON"):
+        dd.read_json("nofile.json", orient="split", lines=True)
+
+
+def test_write_orient_not_records_and_lines():
+    with pytest.raises(ValueError, match="Line-delimited JSON"):
+        dd.to_json(df, "nofile.json", orient="split", lines=True)
```

Comment:
I don't think it really matters what gets passed here, since the `ValueError` is raised before anything is done anyways (I think it's in the first couple of lines of the function). Is there a preferred "it doesn't matter what this value is" value? I used `df` since that already existed, so it didn't have to spend any time creating something new.

Reply:
I suppose I could use `None` and leave a comment?

Reply:
Ah, I see now that `ddf` is already created at the module level. Switched!